### PR TITLE
chore: Update README.md for release version 0.3.0-beta

### PR DESCRIPTION
**Pull request by dvandra**
Adding **Common Errors and FAQ** to suggest easy and quick resolution.

Screenshot from 2020-12-09 15-19-35

### CODE REVIEW

**Comment by arajkumar** [comment]
@dgpatelgit @dvandra @prashbnair, shall we point to this page from vscode error prompt?

File: README.md [hunk 1]
```diff
@@ -98,6 +98,26 @@ providing Insights(security, licenses, AI based guidance) for applications and h
 
 - [GitHub Organization](https://github.com/fabric8-analytics)
 
+# Common Errors and FAQ:
```
Comment (arajkumar) [inline README.md:101]:
```suggestion
# Common Errors and FAQ
```

File: README.md [hunk 2]
```diff
@@ -98,6 +98,26 @@ providing Insights(security, licenses, AI based guidance) for applications and h
...
+
+    **Suggestion:** Resolve the error thrown by `go mod tidy` and `go list` for the project and try again. 
+
+3. Stack analysis fails after applying the recommended version.
+
+    **Suggestion:** Not all recommended versions work directly, developers need to resolve dependencies for the recommended version using `go mod tidy` or manually.
+
+4. Plugin does not highlight vulnerabilities in vendor code.
+
+    **Suggestion:** Currently plugin does not support vendor (or local) code vulnerabilities scanning. Suggest to use `module@version` for dependencies instead of local code.
```
Comment (arajkumar) [inline README.md:119]: @dgpatelgit What will happen if project has go.mod and vendor?

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (arajkumar) [inline README.md:119]:
```suggestion
    **Suggestion:** Currently plugin does not support vendor (or local) code vulnerabilities scanning. Switch to go.mod to express your project dependencies.
```

Comment (dgpatelgit) [reply README.md:119]: @arajkumar We consider only module and version present in go.mod and ignore vendor folder.

**Review by arajkumar — COMMENTED** [review]
lgtm

**Review by arajkumar — APPROVED** [review]
lgtm

**Review by dgpatelgit — APPROVED** [review]
lgtm

**Pull request merged**